common/on-chip-calib.cpp

```diff
@@ -1160,8 +1161,21 @@ namespace rs2
                     _sub_color->ui = *_ui_color;
                     _ui_color.reset();
                 }
+
+                //Restore options that were changed during the calibration.
+                //When calibration is successful options are restored in autocalib_notification_model::draw_content()
```

Comment:
Please check that even if we click on `dismiss`  after calibration the options are restored

Reply:
Checked